### PR TITLE
First draft of sequence diagrams

### PR DESCRIPTION
**Pull request by mikebowyer**

### CODE REVIEW

File: sequence_diagrams/imaged_pushed_to_API_Sequence.wsd [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+control     Lambda_Upload_Function
+control     AWS_Rekognition
+database    AWS_S3
+control     DynamoDB_BackEnd
+database    AWS_DynamoDB
+
+Dashcam_Owner -> Image_Upload_API: {Image, Latitude, Longitude}
+Image_Upload_API -> Lambda_Upload_Function: {Image, Latitude, Longitude}
+Lambda_Upload_Function --> AWS_Rekognition: {Image}
+AWS_Rekognition --> AWS_Rekognition: Detect Conecnts ofImage
```
Comment (tsteiner4) [inline sequence_diagrams/imaged_pushed_to_API_Sequence.wsd]: Detect Contents of Image? Typo I think

**Comment by ktopolov** [comment]
I've never used the review feature; not sure if I have to hit a button to say I looked it over

**Review by tsteiner4 — CHANGES_REQUESTED** [review]
See comment, possible typo on line 15 in imaged_pushed_to_API_Sequence

**Comment by tsteiner4** [comment]
> I've never used the review feature; not sure if I have to hit a button to say I looked it over

On the files changed tab, you can click Review Changes and then select "Approve"

**Comment by ktopolov** [comment]
Worth checking the diagram; it says that the image will be stored in S3 with the bounding box, while the labels and initial metadata are stored in Dynamo.

Shouldn't the bounding box be stored with the rest of the metadata in Dynamo? Not sure how we would store the image with the bounding box, unless we impose the box on top and re-save it. But then we lose the original image

**Review by ktopolov — APPROVED** [review]

File: sequence_diagrams/imaged_pushed_to_API_Sequence.wsd (hunk 1 above)
Comment (mikebowyer) [reply sequence_diagrams/imaged_pushed_to_API_Sequence.wsd]: Yep type, it was resolved in my latest commit.

**Comment by mikebowyer** [comment]
@ktopolov , I think you raise a good point. My perception was that we can just store the single image with any bounding boxes on it, that way we don't have to store the image twice (with and without bounding box). But We could certainly store bounding box information in the DynamoDB as well and then render a bounding box whenever we desired. I am open to both solutions. 
@tsteiner4 do you have preference here?

**Comment by tsteiner4** [comment]
> @ktopolov , I think you raise a good point. My perception was that we can just store the single image with any bounding boxes on it, that way we don't have to store the image twice (with and without bounding box). But We could certainly store bounding box information in the DynamoDB as well and then render a bounding box whenever we desired. I am open to both solutions.
> @tsteiner4 do you have preference here?

It is a lot easier to source an image from a URL than trying to add a box onto it as well. [Link](https://blog.mturk.com/tutorial-generating-images-with-bounding-boxes-on-the-fly-using-aws-lambda-690d2301a5ff) shows how to make a lambda function to do this, so that is probably how I would do it. I can do this in the html for the file if that is preferred to save storage.

**Pull request merged**